site/en/setup.md

```diff
@@ -167,6 +178,8 @@ make all OCRD_MODULES="core ocrd_tesserocr ocrd_cis" # Will install only ocrd_te
 
 (Custom choices for `OCRD_MODULES` and other control variables (cf. `make help`) can also be made permanent by writing them into `Makefile.local`.)
 
+**Note: Never run `make all` as root!
```

Comment:
Isn't that a little bit harsh? I just did that yesterday to test something with ocrd_all in a chroot environment. Nothing bad happened, it did what was expected.

Reply:
... "unless you know exactly what you're doing!"?